docs/src/main/asciidoc/Installation_Guide.adoc

```diff
@@ -0,0 +1,85 @@
+[[Installation_Guide]]
+= Installation Guide
+WildFly team;
+:revnumber: {version}
+:revdate: {localdate}
+:toc: macro
+:toclevels: 3
+:toc-title: Installation Guide
+:doctype: book
+:icons: font
+:source-highlighter: coderay
+:wildflyVersion: 14
+
+:leveloffset: +1
+
+ifndef::ebook-format[:leveloffset: 1]
+
+(C) 2020 The original authors.
+
+ifdef::basebackend-html[toc::[]]
+:numbered:
+
+[[Zipped_Installation]]
+= Installing WildFly from a zipped distribution
+
+That is the historical way to install a complete WildFly server with support for standalone and domain modes. A WildFly distribution
+contains a large number of default configurations allowing you to select the server features and operating modes.
+
+WildFly distribution based installation is well suited when:
+
+* You want to rely on a traditional Jakarta EE application deployment.
+* You want a Jakarta EE compliant server that offers all Jakarta EE features.
+* You are not (yet) concerned by server installation size and memory footprint.
+* You are not yet sure of the kind of operating mode and application you are targeting.
+* Your server instances will contain one or more application deployments.
+
+If that is the kind of installation you are aiming at, the guides that you should read next are:
+
+* link:Getting_Started_Guide{outfilesuffix}[Getting Started Guide] shows you
+how to install, start and configure the server.
+* link:Getting_Started_Developing_Applications_Guide{outfilesuffix}[Getting
+Started Developing Applications Guide] shows you how to build Jakarta EE
+applications and deploy them to WildFly.
+
+[[Galleon_Provisioning]]
+= Installing WildFly with Galleon
+
+Galleon provisioning tooling allows you to construct a customized WildFly installation according to your application needs. 
+Some applications don't need a fully featured server supporting all operating modes and containing all Jakarta EE capabilities. 
+Galleon tooling allows you to select the server capabilities you want to see installed. 
+
+A WildFly server provisioned with Galleon is well suited when:
```

Comment:
Another benefit of Galleon is the ability to update an installation to the latest version, a common problem we are seeing is users asking for help using a version of WildFly 5 or more major releases back!  

TBH for those that choose to use WildFly it really is advisable to routinely keep up to date with the latest to have the latest fixes - often we see end users who get advised to update and because they have left it so long it now becomes a big job for them.

Reply:
Thank-you, yes good point. I added it.